api/src/ol/ol.module.ts

```diff
@@ -147,6 +143,16 @@ for (const role of roles) {
     ...workers,
   ],
   controllers: [OlController],
-  exports: [OlService, TransformerService, Types.ICommunityWalletsService],
+  exports: [OlService, TransformerService, Types.ICommunityWalletsService, BullBoardService],
 })
-export class OlModule {}
+export class OlModule {
+  isProduction = process.env.NODE_ENV === 'production';
+
+  constructor(private readonly bullBoardService: BullBoardService) {
+    if (!this.isProduction) {
+      // Setup BullBoard for the queues in local development environment
+      // Utilize the same queue array to ensure consistency
+      this.bullBoardService.setupBullBoard(queues);
```

Comment:
you can move this in the `bullBoardService` in the [`onModuleInit` method](https://docs.nestjs.com/fundamentals/lifecycle-events#usage).